src/slides/functional-programming-reduce/reduce-example2.js

```diff
@@ -1,18 +1,19 @@
-const myList = ['John', 'Jacob', 'Jingleheimer', 'Schmidt'];
+const myList = ['John', 'Jane', 'Joe', 'Sara'];
 
```

Comment:
This file had way too many comments - it was hard to read the code. Tried to trim it down a bit